core/src/replay_stage.rs

```diff
@@ -2635,6 +2635,7 @@ impl ReplayStage {
                         &bank.rewards,
                         Some(bank.clock().unix_timestamp),
                         Some(bank.block_height()),
+                        bank.transaction_entries_count(),
```

Comment:
Is this really the Bank api that we want? I don't think geyser streams entries, or knows how many entries are in a block, does it?

Reply:
my misunderstanding then, I am looking for the transaction count: all transaction executed success or failure

Reply:
Is transaction_count + transaction_error_count the right one?

Reply:
You know, I'm actually not sure we have a Bank field for that. `Bank::transaction_count()` is closest, but its cumulative across all Banks since slot 0. Let me dig a little.

Reply:
For now, the only way to get the number you want is to get the difference between `bank.transaction_count()` and that method on the parent Bank. Just had a little discussion about adding a counter to Bank: https://discord.com/channels/428295358100013066/439194979856809985/1054878287744217209

Reply:
transaction_count is only the count for transactions 

    /// The number of transactions processed without error
    transaction_count: AtomicU64,

Are we missing transaction landed in error status?

Reply:
Hmm, yes, I guess we are. Looks like this hasn't been activated anywhere yet: https://github.com/solana-labs/solana/pull/22802
I'm not 100% sure that the number you're looking for is currently `transaction_count + transaction_error_count`, but probably pretty easy to spot check.

Reply:
Yeah sorry Lijun, looks like I misled you with this `transaction_entries_count`, should be using the `transaction_count`